surfsense/scripts/health-check.sh

```
@@ -0,0 +1,63 @@
+set -a
+source ./.env
+set +a
+
+FRONTEND_PORT="${FRONTEND_PORT:-3929}"
+BACKEND_PORT="${BACKEND_PORT:-8929}"
+ZERO_CACHE_PORT="${ZERO_CACHE_PORT:-5929}"
+SEARXNG_PORT="${SEARXNG_PORT:-}"
+
+command -v docker >/dev/null 2>&1 || { echo "[ERROR] docker not installed" >&2; exit 1; }
+
+echo "\n=== Docker Compose Services ==="
```

Shellcheck (0.11.0)

[info] echo may not expand escape sequences. Use printf.

(SC2028)